layouts/partials/footer.html

```diff
@@ -42,7 +43,9 @@
               <li class="sitemap-head" style="padding-top: 30px;">Other Projects</li>
               <li><a href="https://libp2p.io/">libp2p</a></li>
               <li><a href="https://ipld.io/">IPLD</a></li>
+              <li><a href="https://drand.love/">Drand</a></li>
               <li><a href="https://multiformats.io/">Multiformats</a></li>
+              <li><a href="https://docs.testground.ai/">Testground</a></li>
```

Comment:
```suggestion
              <li><a href="https://testground.ai/">Testground</a></li>
```
Let's drop the explicitly specified subdomain to futureproof this for when it has top-level content and doesn't just redirect straight to docs 😊

Reply:
Smart, thank you!